src/dbnode/digest/writer.go

```diff
@@ -109,3 +112,42 @@ func (w *fdWithDigestContentsWriter) WriteDigests(digests ...uint32) error {
 	}
 	return nil
 }
+
+// CloseAll closes all the provided writers sequentially and returns
+// a MultiError with any errors encountered.
+func CloseAll(writers ...FdWithDigestWriter) error {
+	var multiErr xerrors.MultiError
+	for _, writer := range writers {
+		if err := writer.Close(); err != nil {
+			multiErr = multiErr.Add(err)
+		}
```

Comment:
nit; `mutliErr.Add` already verifies that the error is not nil so this is not strictly necessary